File: test/aidbox_sdk/snapshots/hl7-fhir-r4-core/Linkage.cs
using Aidbox.FHIR.Base;
using Aidbox.FHIR.Utils;

namespace Aidbox.FHIR.R4.Core;

public class Linkage : DomainResource, IResource
{
    public required Linkage_Item[] Item { get; set; }
    public bool? Active { get; set; }
    public Base.ResourceReference? Author { get; set; }

    public class Linkage_Item : BackboneElement
    {
        public required string Type { get; set; }
        public required Base.ResourceReference Resource { get; set; }
    }
}

File: test/aidbox_sdk/snapshots/python/hl7_fhir_r4_core/Linkage.py
from typing import Optional, List
from dataclasses import dataclass, field
from base import DomainResource
from base import Reference
from base import BackboneElement

@dataclass(kw_only=True)
class LinkageItem(BackboneElement):
    type: str
    resource: Reference

@dataclass(kw_only=True)
class Linkage(DomainResource):
    item: list[LinkageItem] = field(default_factory=list)
    active: Optional[bool] = None
    author: Optional[Reference] = None